src/views/Article.vue

```diff
@@ -130,9 +130,18 @@
               :class="citations.length === 0 ? 'no-citation' : ''"
             >
               <div class="block-header">
-                <h2>
-                  {{ block.blockTitle }}
-                </h2>
+                <div class="d-flex justify-content-between">
+                  <h2>
+                    {{ block.blockTitle }}
+                  </h2>
+                  <b-button
+                    v-b-tooltip.hover.bottom.v-secondary="'分享段落'"
```

Comment:
Why required a single quote in the double quote here?

Reply:
Lol tbh I have no idea, going to fix it soon.

Reply:
Has it been fixed?

Reply:
I find it necesary to include single quotes according to the bootstrapvue docs:
![image](https://user-images.githubusercontent.com/39425103/207034003-b295320c-8141-44f8-9aa4-a935a39ff435.png)